extensions/wikia/Forum/templates/Forum_boardMessage.php

```diff
@@ -1,102 +0,0 @@
-<li class="SpeechBubble message <?php echo ( $removedOrDeletedMessage ? 'hide ':'' ) . ( $showRemovedBox ? ' message-removed':'' ); ?> <? echo 'message-' . $linkid ?>" id="<? echo $linkid ?>" data-id="<? echo $id ?>" data-is-reply="<?= $isreply == true ?>" <? if ( $hide ):?> style="display:none" <? endif; ?> >
```

Comment:
@macbre @TK-999 please confirm that these removed templates are not used anywhere